app/test_main.py

```diff
@@ -1,3 +1,53 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_data",
+    [
+        (
+            0,
+            0,
+            [0, 0]
+        ),
+        (
+            14,
+            14,
+            [0, 0]
+        ),
+        (
+            15,
+            15,
+            [1, 1]
+        ),
+        (
+            23,
+            23,
+            [1, 1]
+        ),
+        (
+            24,
+            24,
+            [2, 2]
+        ),
+        (
+            27,
+            27,
+            [2, 2]
+        ),
+        (
+            28,
+            28,
+            [3, 2]
+        ),
+        (
+            100,
+            100,
+            [21, 17]
+        ),
+    ]
```

Comment:
There are repeated test cases that test similar scenarios, such as (14, 14) and (15, 15) both resulting in [0, 0] and [1, 1] respectively. According to the checklist, make sure that your tests don't test the same case multiple times.